server/archive.api.js

```diff
@@ -0,0 +1,52 @@
+const HomebrewModel = require('./homebrew.model.js').model;
+const router = require('express').Router();
+const asyncHandler = require('express-async-handler');
+
+const archive = {
+	archiveApi : router,
+	/* Searches for matching title, also attempts to partial match */
+	findBrews  : async (req, res, next)=>{
+		try {
+			const title = req.query.title || '';
+			const page = parseInt(req.query.page) || 1;
```

Comment:
Enforce a minimum of `1` to avoid shenanigans

`const page = Math.max(parseInt(req.query.page) || 1, 1);`